api/v1beta1/ssp_webhook.go

```diff
@@ -91,3 +104,63 @@ func (r *SSP) ValidateDelete() error {
 func setClientForWebhook(c client.Client) {
 	clt = c
 }
+
+func validatePlacement(ssp *SSP) error {
+	if err := validateOperandPlacement(ssp.Spec.TemplateValidator.Placement); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
This function can be inlined, or simplified to:
```
return validateOperandPlacement(ssp.Spec.TemplateValidator.Placement)
```

Reply:
Yes, I agree, I will change that